ecodev_front/graphs/subplots.py

```diff
@@ -0,0 +1,39 @@
+"""
+Module implementing how to create subplots in plotly
+"""
+from typing import List
+
+import plotly.graph_objs as go
+from plotly.subplots import make_subplots
+
+
+def subplots(rows: int,
+             cols: int,
+             shared_xaxes: bool = True,
+             row_titles: List[str] | None = None,
+             vertical_spacing: float = 0.05) -> go.Figure:
```

Comment:
```suggestion
             spacing: float = 0.05) -> go.Figure:
```

Reply:
nope it's actually the spacing between different graphs in a vertical manner